lib/common-types.ts

```diff
@@ -83,6 +85,14 @@ export interface VersionedLink<T extends string> {
     version: number
   }
 }
+
+export interface EntityPayload<T> {
+  entities: {
+    sys: { type: 'Array' }
+    items: T[]
+  }
+}
+
```

Comment:
Ehhh.... can we instead have a `BaseCollection` interface that just has the `{ sys, items }` bit? The `entities` tacked on feels very specific to Release

Reply:
Sometimes it's better to write everything twice (WET), especially if the shapes may diverge in the future.